src/receive_imf.rs

```diff
@@ -1647,15 +1648,15 @@ async fn create_or_lookup_group(
 /// Apply group member list, name, avatar and protection status changes from the MIME message.
 ///
 /// Optionally returns better message to replace the original system message.
-/// is_partial: whether the message is not fully downloaded.
```

Comment:
Maybe rename it to is_partial_download as in other places? It's difficult to understand what is "partial" w/o reading this doc comment